src/app/sandbox/utils.ts

```diff
@@ -38,8 +38,11 @@ export const encodeTuple = (
 ): ClarityValue => {
   const tupleData = tuple.reduce(
     (acc, tupleEntry) => {
-      const _type = tupleEntry.type;
-      acc[tupleEntry.name] = encodeAbiClarityValue(value[tupleEntry.name].toString(), _type);
+      const optionalType = isClarityAbiOptional(tupleEntry.type);
+      const type = tupleEntry.type;
+      acc[tupleEntry.name] = optionalType
+        ? noneCV()
```

Comment:
Does this mean it will only accept `none`? It should be able to be empty, for `none` **or** contain a value, for a `some` value.

Reply:
Yes I should be including the value if its present